Bukkit/src/main/java/com/plotsquared/bukkit/generator/BlockStatePopulator.java

```diff
@@ -65,7 +65,10 @@ public void populate(final @NonNull World world, final @NonNull Random random, f
             return;
         }
         final ChunkWrapper wrap = new ChunkWrapper(area.getWorldName(), source.getX(), source.getZ());
-        final ScopedQueueCoordinator chunk = this.queue.getForChunk(wrap.x, wrap.z);
+        final ScopedQueueCoordinator chunk = this.queue.getForChunk(wrap.x, wrap.z,
+                com.plotsquared.bukkit.util.BukkitWorld.getMinWorldHeight(world),
+                com.plotsquared.bukkit.util.BukkitWorld.getMaxWorldHeight(world)
```

Comment:
I think the code in `ScopedQueueCoordinator` assumes its max is inclusive, so the max here should be inclusive too.

Reply:
ScopedQueuedCoordinator doesn't actually use max x/y/z currently

Reply:
It calculates dx, dy, dz based on min/max in its constructor and then uses those

Reply:
it used to be inclusive, so is currently okay (given latest commits)